airbyte-integrations/connectors/source-hubspot/source_hubspot/manifest.yaml

```diff
@@ -508,13 +508,76 @@ definitions:
         class_name: source_hubspot.components.MigrateEmptyStringState
         cursor_field: updatedAt
 
+  contact_lists_stream:
+    $ref: "#/definitions/stream_base"
+    name: contact_lists
+    primary_key: listId
+    retriever:
+      $ref: "#/definitions/base_retriever"
+      requester:
+        $ref: "#/definitions/base_requester"
+        path: /crm/v3/lists/search
+        http_method: POST
+      record_selector:
+        schema_normalization: Default
+        extractor:
+          type: DpathExtractor
+          field_path:
+            - lists
+      paginator:
+        type: DefaultPaginator
+        page_token_option:
+          type: RequestOption
+          inject_into: body_json
+          field_name: offset
+        page_size_option:
+          type: RequestOption
+          inject_into: body_json
+          field_name: count
+        pagination_strategy:
+          type: CursorPagination
+          page_size: 1000
+          cursor_value: "{{ response.get('offset') }}"
+          stop_condition: "{{ not response.get('has-more') }}"
+    incremental_sync:
+      type: DatetimeBasedCursor
+      cursor_field: updatedAt
+      datetime_format: "%ms"
+      lookback_window: P{{ config.get('lookback_window', 0) }}D
+      is_client_side_incremental: true
+      cursor_datetime_formats:
+        - "%ms"
```

Comment:
Small adjustment, but I think we should stick to just emitting state as the string timestamp moving forward since we're already making this a breaking change.

But since we still need to accept state from the previous format, i think we just need to make `%ms` the last priority format in this list to attempt parsing into.